src/Orleans/Runtime/RequestContext.cs

```diff
@@ -42,6 +41,11 @@ public static class RequestContext
         internal const string ORLEANS_REQUEST_CONTEXT_KEY = "#ORL_RC";
         internal const string PING_APPLICATION_HEADER = "Ping";
 
+#if NETSTANDARD_TODO
```

Comment:
The purpose of `NETSTANDARD_TODO` constant is to have something that needs addressing ASAP. So this PR should probably change these to just `NETSTANDARD`; meaning that it is a different implementation for the other platform, but does not need addressing any further (other than at some point we'll get rid of all `NETSTANDARD` conditional compilation entirely too and have 1 way of doing it, for example when we support 4.6, or move platform-specific code to a platform-specific project, but this is longer term)